### DISCUSSION TITLE
Can we install micronaut on linux systems without sdkman

### DISCUSSION THREAD

**Discussion by IshaanDesai45** [micronaut-projects/micronaut-core#8840] [Q&A]
Is sdk man the only option to configure micronaut on linux env ? Apart from this I don't want to clone the directory and build from there.....is there something simulating the donwload with binary option in windows for linux

**Comment by alvarosanchez** [comment]
Assuming that by "install Micronaut" you mean the Micronaut CLI, there are other alternatives:

1. Download the ZIP from the [releases](https://github.com/micronaut-projects/micronaut-starter/releases). For example: https://github.com/micronaut-projects/micronaut-starter/releases/download/v3.8.6/mn-linux-amd64-v3.8.6.zip contains the Linux binary.
2. Use [Homebrew](https://micronaut-projects.github.io/micronaut-starter/latest/guide/index.html#installHomebrew)
